src/test/java/com/hocs/test/glue/decs/BaseStepDefs.java

```diff
@@ -194,6 +194,7 @@ public void clickTheLink(String link) {
     public void iClickTheButton(String buttonLabel) {
         try {
             clickTheButton(buttonLabel);
+            waitABit(500);
```

Comment:
why have we added this? This would add half a second to almost all button presses throughout the regression run.
We should avoid implicit waits as best we can (sometimes they are required), but I think whatever this was added to enable, we would be better off setting an explicit wait at a higher level.